build-system/tasks/karma.conf.js

```diff
@@ -40,6 +41,14 @@ const BABELIFY_CONFIG = Object.assign({}, BABELIFY_GLOBAL_TRANSFORM, {
   sourceMapsAbsolute: true,
 });
 
+// minify-replace babel plugin used to replace experiment constants
```

Comment:
This karma config is used for all karma tests (`gulp unit/integration/a4a`). Do we want to have this plugin for everything or keep it to unit tests only?

Reply:
What is `gulp a4a`? 
But yes, we do want to have this plugin for everything. Integration test should have the plugin if we are running it with compiled code.